addons/sales_team/models/crm_team_member.py

```diff
@@ -15,7 +15,7 @@ class CrmTeamMember(models.Model):
     crm_team_id = fields.Many2one(
         'crm.team', string='Sales Team', group_expand='_read_group_crm_team_id',
         default=False,  # TDE: temporary fix to activate depending computed fields
-        check_company=True, index=True, ondelete="cascade", required=True)
+        index=True, ondelete="cascade", required=True)
     user_id = fields.Many2one(
         'res.users', string='Salesperson',  # TDE FIXME check responsible field
         check_company=True, index=True, ondelete='cascade', required=True,
```

Comment:
:thinking: does this even do anything if company is related to the user's company in the first place? 
I guess maybe for overrides :shrug: